### DISCUSSION TITLE
HW Requirements for latest Firmware

### DISCUSSION THREAD

**Discussion by cmweaver** [mriscoc/Ender3V2S1#597] [Q&A]
The latest firmware requirements state:

"A compatible control board: Creality 4.2.2, 4.2.3, 4.2.7, V24S1_301F1, V24S1_301F4 or SKR Mini E3_V3 for Ender series printers with a STM32 SoC."

I have an Ender 3 S1 Pro that is **v24S1_301**, **_NOT_** v24S1_301F1, v24S1_301F4, as stated in the requirements, is my unit still compatible with the latest firmware version; **Ender3S1-xx-xxxxx**, 12-22-22 release? Is the **F4** at the end of v24S1_301 intended to highlight that the board ALSO has the STM32F401 chip?

Further, my unit has the RCT6 STM32F401 chip, but the docs mention a RCT6 STM32FX, note absence of the F401 from my chip.

**Comment by mriscoc** [comment]
F1 is for the STM32F103 and F4 is for the STM32F401 SoC
